From 7ce0d82703768975ddf7ba2071da957012ccf0e2 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 23 Jul 2018 09:50:52 -0500
Subject: [PATCH] Fix path of crio.service's ExecStart crio binary

crio is installed in /usr/bin not /usr/local/bin so update the
crio.service file's ExecStart line to reflect that.

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
 contrib/systemd/crio.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/systemd/crio.service b/contrib/systemd/crio.service
index 35d6d42..99724d8 100644
--- a/contrib/systemd/crio.service
+++ b/contrib/systemd/crio.service
@@ -8,7 +8,7 @@ Type=notify
 EnvironmentFile=-/etc/sysconfig/crio-storage
 EnvironmentFile=-/etc/sysconfig/crio-network
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/local/bin/crio \
+ExecStart=/usr/bin/crio \
           $CRIO_STORAGE_OPTIONS \
           $CRIO_NETWORK_OPTIONS
 ExecReload=/bin/kill -s HUP $MAINPID
-- 
2.18.0

